package.json

```diff
@@ -115,6 +115,7 @@
 		"hpagent": "1.2.0",
 		"react-markdown": "^9.0.1",
 		"semver": "^7.6.0",
+		"strip-ansi": "^7.1.0",
```

Comment:
This was necessary to strip ANSI that comes in the output from CLI - otherwise, the Error and Success messages contain ANSI characters and manually removing them is quite tedious as it is hard to match all of them. I felt like this was appropriate to use